### PR TITLE
Gradle upgrade

### PR DESCRIPTION
**Pull request by ajayk**
Fixes # .

### Did you remember to?

- [ ] Add test case(s)
- [x] Update `CHANGES.txt`

We encourage pull requests that:

* Add new features to TestNG (or)
* Fix bugs in TestNG

If your pull request involves fixing SonarQube issues then we would suggest that you please discuss this with the 
[TestNG-dev](https://groups.google.com/forum/#!forum/testng-dev) before you spend time working on it.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Current
+New: Upgrade to gradle5
```
Comment (krmahadevan) [inline CHANGES.txt]: Can you please include a reference to https://github.com/cbeust/testng/issues/1968 which talks about this upgrade ?

Comment (krmahadevan) [reply CHANGES.txt]: @ajayk - Fair enough.

**Review by krmahadevan — APPROVED** [review]

**Review by juherr — APPROVED** [review]
Thank a lot. It fixes the most important part of #1968 👍 

It will be awesome if you can check the OSGI part.
We'd love to switch to the Kotlin description too.

Maybe another pull-requests? :)

**Comment by cbeust** [comment]
+1 to converting to the Kotlin DSL. I started working on this but ended up in a documentation maze that quickly made me give up.

**Comment by krmahadevan** [comment]
@cbeust - any words of wisdom as to where can I start if I were to take this up ? Can be a good learning experience but I want it to be targeted :)

**Pull request merged**